### PR TITLE
CACTUS-732 :: Migrate to Yarn Berry

### PR DESCRIPTION
**Pull request by Dhalton**
Ticket & UAT: https://repayonline.atlassian.net/browse/CACTUS-732

I removed that `tsconfig.json` like we talked about in chat because I'm pretty sure it wasn't doing anything for us,  but if you guys notice anything strange as a result, let me know.

Also, CircleCI hasn't been able to check out the repo. I made a ticket to switch to Github Actions: https://repayonline.atlassian.net/browse/CACTUS-989

### CODE REVIEW

File: modules/testing-tools/package.json [hunk 1]
```diff
@@ -5,9 +5,7 @@
   "main": "src/index.js",
   "repository": "https://github.com/repaygithub/ui-tools/tree/master/modules/testing-tools",
   "license": "MIT",
-  "bin": {
-    "testing-tools": "bin/testing-tools"
-  },
+  "bin": "bin/testing-tools",
```
Comment (Dhalton) [inline modules/testing-tools/package.json:8]: I have no idea why yarn made this change here but not in `repay-scripts`. I've tried reverting this, but yarn just changes it again whenever you install. Both CLI tools worked for me when I published them with `yalc`, so... 🤷‍♂️

File: modules/repay-scripts/tests/test-setup.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ class TestSetup {
 
   async setup() {
     await fs.copy(path.join(__dirname, 'fixtures', this.fixture), this.tempDir)
+    await exec('yarn set version 1.22.0', { cwd: this.tempDir })
```
Comment (maxrcollins) [inline modules/repay-scripts/tests/test-setup.js:29]: :D

File: modules/testing-tools/package.json (hunk 1 above)
Comment (maxrcollins) [reply modules/testing-tools/package.json:8]: huh. weird.

**Review by maxrcollins — APPROVED** [review]
Looks good to me. I'll have to look into that "bin" thing later

File: modules/repay-scripts/tests/test-setup.js (hunk 2 above)
Comment (Dhalton) [reply modules/repay-scripts/tests/test-setup.js:29]: Yeah, several hours of debugging just for a one-liner to do the trick. Good times!

**Review by wilysword — APPROVED** [review]

**Pull request merged**